lib/services/elasticsearch.js

```diff
@@ -153,519 +166,479 @@ class ElasticSearch extends Service {
         return this.kuzzle.services.list.internalCache.pexpire(cacheKey, ttl);
       })
       .then(() => this.client.scroll(esRequest))
-      .then(result => formatSearchResults(result))
-      .catch(error => Bluebird.reject(this.esWrapper.formatESError(error)));
+      .then(({ body }) => {
+        const apiResponse = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "apiResponse". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1488)